README.md

```diff
@@ -1,4 +1,7 @@
-## RPC Endpoint
+## Flashbots RPC Endpoint (_flashbots-rpc-endpoint_)
+
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+![Discord](https://img.shields.io/discord/755466764501909692)
```

Comment:
If we want to have this Discord badge, I suggest it links to our Discord as well.